src/fhirtypes/ElementDefinition.ts

```diff
@@ -319,10 +319,20 @@ export class ElementDefinition {
   }
 
   validate(): ValidationError[] {
+    const errors: ValidationError[] = [];
     if (this.slicing) {
-      return this.validateSlicing(this.slicing);
+      if (this.sliceName) {
+        errors.push(
+          new ValidationError(
+            'An element with a slice name should not define its own slicing. Instead, define additional discriminators to the original slicing on the base element.',
```

Comment:
You took my suggestion, but it turns out my suggestion was a tiny bit awkward. Sorry. I think this reads better:
```suggestion
            'An element with a slice name should not define its own slicing. Instead, append additional discriminators to the original slicing on the base element.',
```